concourse/steps/release.py

```diff
@@ -338,6 +347,41 @@ def _push_tag(tag):
         else:
             raise NotImplementedError
 
+        if self.publishing_policy == ReleaseCommitPublishingPolicy.TAG_AND_MERGE_BACK:
```

Comment:
codestyle: enum-values should be compared by identity (`is`) rather than equality (`==`)

Reply:
overall lgtm